src/app/features/shows/pages/movie-list/movie-list.component.html

```diff
@@ -7,8 +7,8 @@ <h1 class="mat-headline-1">Movies</h1>
     <input matInput placeholder="eg. Avenger" [(ngModel)]="searchText" (ngModelChange)="this.searchTextUpdate.next($event)">
   </mat-form-field>
 </div>
-<span class="mat-subtitle-2"> {{ movies === undefined ? 0 : movies.length }} items</span>
-<div class="list" *ngIf="movies === undefined; else yes">
+<span class="mat-subtitle-2"> {{ loading === true ? 0 : movies.length }} items</span>
+<div class="list" *ngIf="loading === true; else yes">
```

Comment:
```suggestion
<div class="list" *ngIf="loading; else yes">
```
boolean hết mà so chi cực vậy e